### DISCUSSION TITLE
不支持银行卡

### DISCUSSION THREAD

**Discussion by xc-bnu** [getlantern/lantern-forum-cn#372] [Q&A]
使用银行卡支付时出现：不支持您的银行卡。
如何解决这个问题，可以支付宝、微信支付吗？
![image](https://github.com/getlantern/lantern-forum-cn/assets/32566893/224c0bf4-bf4b-47ca-9ec5-bd00e0ff1013)

**Comment by xc-bnu** [comment]
@Mike777Khoo有什么解决方案吗

**Reply by MaxMKMax** [reply]
https://buy.stripe.com/eVacPZe3A5cp1oc4gO
您好. 这是我们的另一个购买渠道. 可以用支付宝付费的. 谢谢您支持蓝灯!

**Reply by xc-bnu** [reply]
通过你给的购买渠道购买完后，lantern网页没有显示续期成功

**Reply by MaxMKMax** [reply]
@xc-bnu 好的. 我来帮您问客服一下. 看是否需要手动升级. 请问您可以给我支付单据吗? 谢谢.

**Reply by MaxMKMax** [reply]
@xc-bnu 由于支付网关提供商的问题, 客服说信用卡暂时无法使用. 您可以用支付宝来续费.

**Reply by MaxMKMax** [reply]
@xc-bnu 您刚才的信用卡有成功扣款吗? 你查看一下. 也顺便提供我您的账号邮箱.

**Reply by xc-bnu** [reply]
@Mike777Khoo我使用的是支付宝付款的，已经成功扣款，我账号邮箱2491730877@qq.com

**Reply by xc-bnu** [reply]
付款成功后，lantern页面仍然显示
![image](https://github.com/getlantern/lantern-forum-cn/assets/32566893/98731ffc-ffef-4f04-9bc8-2f39254397cd)

**Reply by MaxMKMax** [reply]
@xc-bnu 好的. 我来帮您反馈. 看是否可以手动帮您升级! 但要等到深夜支付团队才会上线了. 谢谢您的等待!

**Reply by xc-bnu** [reply]
@Mike777Khoo支付问题还没得到解决
![image](https://github.com/getlantern/lantern-forum-cn/assets/32566893/e93bc5c7-6a85-436e-8b5b-d9982a016d3b)

**Reply by xc-bnu** [reply]
@Mike777Khoo支付问题仍然没有得到解决，同时latern网页提示：
![image](https://github.com/getlantern/lantern-forum-cn/assets/32566893/ae9a2f43-ba45-42cf-9453-38c951df159d)
都好几天了，还请尽快解决。

**Reply by xc-bnu** [reply]
@Mike777Khoo已经一周了，我还没有得到回复，请尽快回复，是否已经解决这个问题。

**Reply by xc-bnu** [reply]
@Mike777Khoo 为什么一直得不到回复，我想知道问题是否已经得到解决了？

**Reply by xc-bnu** [reply]
@Mike777Khoo 想通过邮箱登录lantern，一直收不到授权码。试了好多次，都收不到授权码。
![image](https://github.com/getlantern/lantern-forum-cn/assets/32566893/0cbd58ac-0d1f-4cd5-befe-a89c5c25bd50)

**Comment by MaxMKMax** [comment]
@xc-bnu 请问您是通过哪一个地方或页面购买的?

**Reply by xc-bnu** [reply]
就是lantern网页版里面的续费界面

**Reply by MaxMKMax** [reply]
@xc-bnu 好的. 请您放心. 我已经在帮您问着了. 等付款团队答复我. 我再回复您.

**Comment by MaxMKMax** [comment]
@xc-bnu 您好. 很抱歉我迟回复您了. 看来您的邮件收不到我们的邮件. 我已让客服给您发邮件了. 让我再帮你获取验证码. 但需要等到星期二了. 非常抱歉啊. 谢谢您支持蓝灯!

**Reply by xc-bnu** [reply]
@Mike777Khoo不仅是收不到验证码，登录不上lantern，同时我在4月3号支付617元后，不确定lantern有没有续期成功。4月3号到现在，一直都没有给我解决方案。

**Reply by MaxMKMax** [reply]
@xc-bnu 您好. 支付团队已经给您发激活码了. 请查看邮箱. 因为您的购买没有升级成功所以收不到验证码. 但您放心, 您的付费已经成功. 所以您用这个码就能登陆了!

**Comment by MaxMKMax** [comment]
@xc-bnu 您好，谢谢您的反馈。非常抱歉影响了您的体验感。谢谢您的等待与支持。

您可以帮助我们在应用程序中提交报告您的问题吗？您可以访问Lantern应用程序并提交log/ 报告。

1. go to Support (转到支持)
2. select Report an Issue (选择报告问题)
3. choose issue type (选择问题类型)
4. include additional details (包括额外的详细信息)
5. hit Send Report (点击发送报告)

当您在app提交问题报告后您会收到邮件. 邮件标题上有 8 位数号码. 请回来这里留言您的log/ticket号码. 因为我需要拿您的反馈报告号码来和科技团队沟通您遇到的问题. 不然我无法提交您遇到的问题给科技团队.

有了这些资料，我们科技团队便可以尽快解决问题。您可以在留言区给我知道。谢谢。
我们将非常感谢您。

**Comment by MaxMKMax** [comment]
@xc-bnu 好的. 完全了解您遇到的情况. 下周我会和技术团队开会, 我会和他们谈您的问题. 但我需要您的报告票号. 所以希望您能够帮我提交以下应用内报告. 麻烦您了, 谢谢您!

**Reply by xc-bnu** [reply]
@Mike777Khoo报告票号如何获取

**Reply by xc-bnu** [reply]
@Mike777Khoo让我先收到验证码，登录上lantern。我是不是才能提交我的问题。

**Reply by xc-bnu** [reply]
我现在一直收不到验证码，登录不上lantern

**Reply by MaxMKMax** [reply]
@xc-bnu 嗯嗯. 我会让支付团队帮您查. 星期一晚上他们上线了我帮您问.

**Reply by xc-bnu** [reply]
@Mike777Khoo能否让我先登上lantern，我已经两周登不上lantern了。2491730877@qq.com邮箱一直收不到验证码。

**Reply by MaxMKMax** [reply]
@xc-bnu 好的. 我来帮您问付款团队看是否可以尽快给您提供验证码. 请您稍等. 我帮您要求紧急申请了!

**Comment by MaxMKMax** [comment]
@xc-bnu 您好. 支付团队已经给您发激活码了. 请查看邮箱. 因为您的购买没有升级成功所以收不到验证码. 但您放心, 您的付费已经成功. 所以您用这个码就能登陆了!

**Reply by xc-bnu** [reply]
@Mike777Khoo 已经登上了，谢谢，但是网页上说续费2年赠送3个月，但是我这个续费完并没有赠送3个月，只有2年。
![image](https://github.com/getlantern/lantern-forum-cn/assets/32566893/8386b99a-b2c2-4cb9-93cf-1e0215390cf8)

**Reply by MaxMKMax** [reply]
@xc-bnu 
嗯嗯好的. 我帮您通知付款团队让他们帮您更新配套. 不好意思啊. 谢谢!

**Reply by MaxMKMax** [reply]
@xc-bnu 您好. 您的专业班已经增加 3 个月了. 谢谢您支持蓝灯!

**Reply by xc-bnu** [reply]
@Mike777Khoo 好的，谢谢
